app/src/main/java/com/github/se/icebreakrr/ui/map/MapView.kt

```diff
@@ -322,6 +378,37 @@ fun MapScreen(
       }
 }
 
+/**
+ * Creates a bitmap for the user marker with a blue circle and a white border.
+ *
+ * @return BitmapDescriptor for the user marker.
+ */
+private fun createUserMarkerBitmap(): BitmapDescriptor {
+  // Create a bitmap for the blue circle with a white border programmatically
+  val circleBitmap =
+      Bitmap.createBitmap(60, 60, Bitmap.Config.ARGB_8888) // Increased size for border
+  val canvas = Canvas(circleBitmap)
+
+  // Draw the white border
+  val borderPaint =
+      Paint().apply {
+        color = android.graphics.Color.BLACK // White color for the border
+        isAntiAlias = true
+        style = Paint.Style.FILL // Fill the circle
+      }
+  canvas.drawCircle(30f, 30f, 30f, borderPaint) // Draw the border circle
+
+  // Draw the blue circle
+  val bluePaint =
+      Paint().apply {
+        color = android.graphics.Color.parseColor("#1FAEF0") // IceBreakrr Blue
```

Comment:
`color = android.graphics.Color.parseColor("#1FAEF0") // IceBreakrr Blue`
nitpick: I think we declared a project constant for this color.

Reply:
Yep need to change that